src/Commands/core/CliCommands.php

```diff
@@ -308,6 +308,10 @@ public function makeEntitiesAvailableWithShortClassNames(): void
     {
         foreach ($this->entityTypeManager->getDefinitions() as $definition) {
             $class = $definition->getClass();
+            $reflectionClass = new ReflectionClass($class);
```

Comment:
```suggestion
            $reflectionClass = new \ReflectionClass($class);
```
Should have manually test this earlier. Did now and found out a leading slash is needed.